server.js

```diff
@@ -16,16 +17,16 @@ if (permissiveCors) {
   };
 } else {
   corsOptions = {
-    origin: [
-      'https://rsk.co',
-      /\.rsk\.co$/,
-    ],
+    origin: ['https://rsk.co', /\.rsk\.co$/],
```

Comment:
good thing linter changed this... cos it reminds me that we should add rootstock.io and its subdomains as well (later PR, will let you know when)